misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -65,6 +65,17 @@ internal class BoundAction<A : WebAction>(
     )
   }
 
+  fun matchByUrl(url: HttpUrl): BoundActionMatch? {
+    val patchMather = pathPattern.matcher(url) ?: return null
+    return BoundActionMatch(
+        this,
```

Comment:
Consider using Kotlin named paramters so it's more clear at call site.

Reply:
Fixed